packages/lib/package.json

```diff
@@ -12,7 +12,7 @@
     "tsc": "tsc --project tsconfig.json",
     "watch": "tsc --watch --preserveWatchOutput --project tsconfig.json",
     "generatePluginTypes": "rm -rf ./plugin_types && yarn tsc --declaration --declarationDir ./plugin_types --project tsconfig.json",
-    "test": "jest --verbose=false",
+    "test": "node --security-revert=CVE-2023-46809 ./node_modules/.bin/jest --verbose=false",
     "test-ci": "yarn test"
```

Comment:
This seems to be causing test failures for me in my local development environment:
```console
$ cd packages/lib
$ yarn test
/usr/bin/node-22: Error: Attempt to revert an unknown CVE [CVE-2023-46809]
$ node --version
v22.16.0
```

Perhaps upgrading to Node 22 would be an alternate fix?